### ISSUE TITLE
Schematic v2 support

### ISSUE THREAD

**Issue by N0tMyFaultOG** [IntellectualSites/PlotSquared#2534]
Just a small note I'm leaving here that we should update our schematic reader/writer using v2 within the next release.
It allows you to store [more things](https://github.com/SpongePowered/Schematic-Specification#current-version---2) which is quiet useful for plot schematics.
v2 just came out some months ago and we're still using v1 from the original PlotSquared v4 release.

**Comment by Sauilitired** [comment]
We're not interested in storing the entities, but storing the biomes might be of value. It's a rather low priority addition tho, imo.

**Comment by N0tMyFaultOG** [comment]
> We're not interested in storing the entities (or rather, we haven't been), but storing the biomes might be of value. It's a rather low priority addition tho, imo.

People don't store entities in their schematics (or at least didn't attempt to get support for it yet), but the biome part looks promising.

**Comment by Sauilitired** [comment]
I agree, especially considering we provide tools to change plot biomes and even go as far as including it in the default plot info output.

**Comment by MattBDev** [comment]
I thought we already supported schematic v2 on plots but not on roads?  Either way I'm not opposed to adding support for both entities and biomes.
